benchmark/mbedtls_config.h

```diff
@@ -89,10 +89,6 @@
 #define MBEDTLS_RSA_C
 #endif
 
-#if !defined(MBEDTLS_DHM_C)
-#define MBEDTLS_DHM_C
-#endif
-
```

Comment:
@k-stachowiak: Thanks for the PR, but in my opinion this is not sufficient. I think you should explicitly use #undef if you want that particular feature to be disabled.

Reply:
@andresag01 Thanks for the review.

This change was made based on the following reasoning: there are some features used here, that are not even enabled by default. Therefore, we should not enable them forcibly. Should, in the future, the features become enabled by default (and break this app), we may return to this issue, and e.g. explicitly disable them.

Looking at this from another perspective, there are probably more features in Mbed TLS that would break the benchmark, but we don't explicitly disable the in this file, since they are not enabled by default. Should `DHM` be treated specially in this case?

Please consider my arguments and confirm, if you think the feature should still be explicitly disabled.